browser/src/docstatefunctions.js

```diff
@@ -150,3 +152,61 @@ app.isExperimentalMode = function () {
 		return app.socket.WSDServer.Options.indexOf('E') !== -1;
 	return false;
 };
+
+app.calc.isPartHidden = function (part) {
+	if (!app.map._docLayer || !app.map._docLayer._lastStatusJSON) return false;
```

Comment:
we can add helper: `app.isLoaded = () => { return app.map._docLayer && app.map._docLayer._lastStatusJSON; }` to simplify

Reply:
I can't add it as "app.isLoaded" because it's Calc specific. But that's right, we need a better approach and i will plan to refactor this. Thank you.